metrics/prometheus/prometheus.go

```diff
@@ -134,21 +134,21 @@ func RegisterCallbackGaugeWithLabels(namespace, subsystem, name, help string, co
 	))
 }
 
-type prometheusHistogram struct {
+type prometheusSummary struct {
 	*prometheus.SummaryVec
 	Pairs map[string]string
 }
 
-// NewHistogram returns a new Histogram backed by a Prometheus summary. It
-// uses a 10-second max age for bucketing. The histogram is automatically
-// registered via prometheus.Register.
-func NewHistogram(namespace, subsystem, name, help string, fieldKeys []string) metrics.Histogram {
-	return NewHistogramWithLabels(namespace, subsystem, name, help, fieldKeys, prometheus.Labels{})
+// NewSummary returns a new Histogram backed by a Prometheus summary. It uses
+// a 10-second max age for bucketing, emulating statsd. The histogram is
```

Comment:
I fixed it there because, to the best of my knowledge, it replicates the behavior of statsd as it exports to graphite, and I defer to statsd's opinions. If you think it's worthwhile, I'll add a new constructor that parameterizes the max age.